File: hp-server-golang/hp-server-lib/entity/user_custom_entity.go
package entity

import "time"

type UserCustomEntity struct {
	Id *int `json:"id" gorm:"primaryKey;autoIncrement"`

	Username string `json:"username"`

	Password string `json:"password"`

	Desc string `json:"desc"`

	CreateTime time.Time `json:"createTime"`
}

func (UserCustomEntity) TableName() string {
	return "user_custom"
}

File: hp-server/src/main/java/net/hserver/hplite/domian/entity/UserCustomEntity.java
package net.hserver.hplite.domian.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.util.Date;

@Data
@TableName("user_custom")
public class UserCustomEntity{
    @TableId(type = IdType.AUTO)
    private Integer id;

    private String username;

    private String password;

    private String desc;

    private Date createTime;

}
